### ISSUE TITLE
Please support cssLiterals options

### ISSUE THREAD

**Issue by khuongduybui** [ThisIsManta/stylus-supremacy#8]
The VSCode extension is removing all my `@css {}` blocks.

**Comment by ThisIsManta** [comment]
Thanks for your interested in Stylus Supremacy.

Normally, it should support all the features listed in the official Stylus website. Hence, the lack of CSS literal support could be a bug.

Please bear with me while I'm investigating.

**Comment by ThisIsManta** [comment]
@khuongduybui would you kindly share the `@css {}` block that went missing when format using **Stylus Supremacy**?

**Comment by khuongduybui** [comment]
Ah... you are using @css {} as top-level block.
I'm using them inside other blocks, even inside values, such as

```
body[theme=dark]
  textarea:focus
    border-color @css { ${$design.primaryLight} }
```
When I call "format document" my `@css` is removed from the `border-color` line.

If I rewrite this as
```
@css {
  body [theme=dark] textarea:focus {
    border-color: ${ $design.primaryLight }
  }
}
```
then call "format document", it works as expected.

**Comment by ThisIsManta** [comment]
This means a bug! Please spare me some time to fix this.

**Comment by ThisIsManta** [comment]
This has been fixed in [version 2.2.2](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.2.2) for **NPM** and **VSCode**.
If you are using **Atom**, please wait a few days as I does not own the package.

Let me know if you have any further issues.
